Backend/app/routes/brand_dashboard.py

```
@@ -0,0 +1,1132 @@
+    validate_uuid_format(brand_id, "brand_id")
+    
+    try:
+        # Get brand's campaigns
+        campaigns = safe_supabase_query(
+            lambda: supabase.table("sponsorships").select("*").eq("brand_id", brand_id).execute(),
+            "Failed to fetch campaigns"
+        )
+        
+        # Get brand's profile
+        profile_result = supabase.table("brand_profiles").select("*").eq("user_id", brand_id).execute()
+        profile = profile_result.data[0] if profile_result.data else None
```

Ruff (0.13.3)

Local variable `profile` is assigned to but never used

Remove assignment to unused variable `profile`

(F841)